locale/de-DE.json

```diff
@@ -94,5 +94,14 @@
     "privacy": "Datenschutz",
     "welcome_message": "Hallo {name}",
     "logout": "Abmelden"
+  },
+  "reviews": {
+    "rating": "Bewertung",
+    "no_reviews": "Sei der Erste, der dieses Produkt bewertet:",
+    "review_count": "({count} {count, plural, =1 {Bewertung} other {Bewertungen}})",
+    "button_add": "Bewertung schreiben",
+    "button_all": "Alle {0} Bewertungen",
+    "author": "von {0}",
```

Comment:
Needs to be adjusted to use parameter name instead. See the comment in the appropriate file.

Reply:
Done.